pkg/apis/config/v1alpha1/resourceinterpreterwebhook_types.go

```diff
@@ -56,6 +56,21 @@ type ResourceInterpreterWebhook struct {
 	Name string `json:"name"`
 
 	// ClientConfig defines how to communicate with the hook.
+	// It supports two mutually exclusive configuration modes:
```

Comment:
If both of them are configured, the service takes effect, without verifying failure.

Reply:
Updated, please take another look